pkg/util/fast_int_set.go

```diff
@@ -200,6 +200,22 @@ func (s FastIntSet) Copy() FastIntSet {
 	return c
 }
 
+// CopyFrom sets the receiver to a copy of c, which can then be modified
+// independently.
+func (s *FastIntSet) CopyFrom(c FastIntSet) {
```

Comment:
Why is this useful vs `a = b.Copy()`?

Reply:
It's cheaper from what I recall. The extra allocations were showing in the profile heavily.

Reply:
Oh, I guess you were in the "large" case, where we are reusing the `Sparse` set.

Reply:
Note that `FastIntSet` is intended for cases where usually we are in the "small" case. For usecases where we are usually in the `large` case, using a `Sparse` directly or a slice of bits might be faster.

Reply:
Ok, hmm. In this PR I added it to keep track of the "remaining values cols by idx" - in other words, the row-ordinal for every column that's expected to be in the value part of the keys within a row that we haven't seen so far. This needs to be tracked to figure out what bits to set in the per-column null bitmap.

I think these ordinals should be low in the common case - low being between 0 and 64, right?

Reply:
Yeah, I can't imagine that going over 64 in practice so perhaps the case where you were seeing allocs in `Copy` is worth investigating.

Reply:
(adding CopyFrom is fine with me though)